core/src/test/scala/caliban/TestUtils.scala

```diff
@@ -34,7 +36,13 @@ object TestUtils {
     case class Mechanic(shipName: String) extends Role
   }
 
-  case class Character(name: String, nicknames: List[String], origin: Origin, role: Option[Role])
+  @GQLDirective(Directive("key", Map("name" -> StringValue("name"))))
+  case class Character(
+    @GQLDirective(Directive("external")) name: String,
```

Comment:
Do we want to keep these?

Reply:
Hmm I think I was going to use them in tests but didn't end up doing so...

Reply:
Yeah they are used in the RenderingSpec